data/categories.txt

```diff
@@ -608,7 +608,7 @@ sw:^Hoteli|restorenti|mgahawa|chakula
 amenity-fuel|@fuel
 en:^Gas Station|3fuel|petrol|petrol station|U+26FD
 ru:Бензин|топливо|азс
-ar:^محطة وقود|وقود|غاز
+ar:^محطة وقود|وقود|غاز|سولار|ديزل|بنزين|بترول
```

Comment:
One last comment: note the amenity-fuel|@fuel at Line 608.
All synonyms from @fuel will be appended to amenity-fuel, so there is no need in duplicating them.